build.sh

```diff
@@ -0,0 +1 @@
+ DOCKER_BUILDKIT=1 docker build --rm  -t zkwasm .
```

Comment:
This line could be in README or `build.sh` should be renamed as `docker-build.sh`. I would suggest just add a few lines in README.